packages/manager/modules/anthos/src/dashboard/general-information/general-information.html

```diff
@@ -52,64 +64,84 @@
-                                ></span>
+                    <oui-action-menu
+                        aria-label="Actions"
+                        data-compact
+                        data-placement="end"
+                    >
+                        <oui-action-menu-item
+                            data-on-click="$ctrl.onAnthosRegeneratePasswordClick()"
+                        >
+                            <span
+                                data-translate="anthos_tenant_dashboard_general_information_tile_security_center_access_anthos_menu_regenerate_pwd"
+                            ></span>
                         </oui-action-menu-item>
                     </oui-action-menu>
-
                 </oui-tile-definition>
 
                 <!--Netapp access-->
                 <oui-tile-definition
-                    data-term="{{:: 'anthos_tenant_dashboard_general_information_tile_security_center_access_netapp_title' | translate }}">
-
+                    data-term="{{:: 'anthos_tenant_dashboard_general_information_tile_security_center_access_netapp_title' | translate }}"
+                >
                     <oui-tile-description>
-                        <oui-clipboard class="mb-2" data-model="$ctrl.tenant.storage.accessUrl"></oui-clipboard>
+                        <oui-clipboard
+                            class="mb-2"
+                            data-model="$ctrl.tenant.storage.accessUrl"
+                        ></oui-clipboard>
                         <a
                             class="oui-button oui-button_secondary"
                             data-ng-href="{{$ctrl.tenant.storage.accessUrl}}"
                             target="_blank"
                             rel="noopener"
                         >
-                            <span data-translate="anthos_tenant_dashboard_general_information_tile_security_center_access_netapp_cta_start_console"></span>
+                            <span
+                                data-translate="anthos_tenant_dashboard_general_information_tile_security_center_access_netapp_cta_start_console"
+                            ></span>
                             <span
                                 class="oui-icon oui-icon-external-link"
                                 aria-hidden="true"
                             ></span>
                         </a>
                     </oui-tile-description>
 
-                    <oui-action-menu aria-label="Actions" data-compact data-placement="end">
-                        <oui-action-menu-item data-on-click="$ctrl.onNetappRegeneratePasswordClick()">
-                                <span
-                                    data-translate="anthos_tenant_dashboard_general_information_tile_security_center_access_netapp_menu_regenerate_pwd"
-                                ></span>
+                    <oui-action-menu
+                        aria-label="Actions"
```

Comment:
**nitpick:** Might be great if we can move this to a translation file.

Reply:
This is done